docs/index.rst

```diff
@@ -3,14 +3,27 @@
    You can adapt this file completely to your liking, but it should at least
    contain the root `toctree` directive.
 
-Welcome to Switchboard's documentation!
-=======================================
+Welcome to Switchboard
+======================
 
-Contents:
+Switchboard is a feature flipper library for the Pyramid or Pylons stack
+(including TurboGears). Originally used to selectively roll out changes to the
+SourceForge site, the library lets you easily control whether a particular
+change (a switch) is active.
+
+You can make switches active for a certain percentage of visitors, all
+visitors to a particular host in a cluster, or if a particular string is
+present in the query string. Furthermore you can easily create your own
+conditions to do fancier things like geo-targeting, specific users, etc.
+In short, Switchboard turns you into a continuous deployment ninja.
```

Comment:
It doesn't really do anything with CD but instead with A/B testing and the like.